compiler/luci/pass/src/ForwardReshapeToUnaryOpPass.cpp

```diff
@@ -180,6 +258,100 @@ class ForwardReshape final : public luci::CircleNodeMutableVisitor<bool>
 
     return forward_reshape(reshape, node);
   }
+
+  bool visit(luci::CircleDiv *node)
+  {
+    luci::CircleReshape *reshape = nullptr;
+    luci::CircleConst *const_value = nullptr;
+
+    reshape = dynamic_cast<luci::CircleReshape *>(node->x());
+    const_value = dynamic_cast<luci::CircleConst *>(node->y());
+    if (reshape == nullptr or const_value == nullptr)
+    {
+      reshape = dynamic_cast<luci::CircleReshape *>(node->y());
+      const_value = dynamic_cast<luci::CircleConst *>(node->x());
+    }
+
+    if (reshape == nullptr or const_value == nullptr)
+      return false;
+
+    if (const_value->dtype() != loco::DataType::FLOAT32)
+      return false;
+
+    // Should be scalar
+    if (const_value->size<loco::DataType::FLOAT32>() != 1)
+      return false;
+
+    return forward_reshape(reshape, node, const_value);
+  }
+
+  bool visit(luci::CircleMul *node)
+  {
+    luci::CircleReshape *reshape = nullptr;
```

Comment:
This forward the reshape through the Mul (one of the input is scalar const value)